### ISSUE TITLE
UI: Update the UI of "About App" section

### ISSUE THREAD

**Issue by HeySitam** [HeySitam/CovidHelp#24]
## Description:
- Update the UI of the `-: About App :-` section. Make it more attractive.
## Existing UI :

![about app](https://user-images.githubusercontent.com/76570320/135729447-4053bd2d-b4c0-4a3b-80ca-6f42173ed2aa.jpg)

## PR Instruction:
- Issue will be assigned on a first come first serve basis.
- If you are thinking to make a pr then at first be sure to get assigned.
- before pushing your PR please make sure to fetch upstream.
- your pr should be pushed on the `develop` branch only.
- try to complete your assigned task within 3 to 5 days and if you need more time then please let me know. As the program has limited time only so please try to complete as fast as you can.
- With your PR try to send a screenshot or screen recording of your work

**Comment by TishaJhabak1014** [comment]
@sitamadex11 can I take this issue plz. Would love to work on it.

**Comment by HeySitam** [comment]
sure go for it.

**Comment by TishaJhabak1014** [comment]
@sitamadex11 sir, also plz add `hactoberfest-accepted` label : )

**Comment by HeySitam** [comment]
@TishaJhabak1014 done

**Comment by HeySitam** [comment]
@TishaJhabak1014 Any Updates?

**Comment by HeySitam** [comment]
@TishaJhabak1014  please join our group for discussion. 
[CLICK HERE](https://chat.whatsapp.com/JXYziNoD3lD9RWQwD3VyEe) TO JOIN.

**Comment by TishaJhabak1014** [comment]
> @TishaJhabak1014 Any Updates?

@sitamadex11 I doubt I lack the complete tech stack to fix the issue as I am only familiar with web dev with html, css, js.

**Comment by HeySitam** [comment]
Okk no issues.

**Comment by SnehaKapoo1** [comment]
please assign me this issue I will be very happy to be merged with my attractive UI idea.

**Comment by HeySitam** [comment]
sure you can start working.
@SnehaKapoo1  please join our group for discussion. 
[CLICK HERE](https://chat.whatsapp.com/JXYziNoD3lD9RWQwD3VyEe) TO JOIN.

**Comment by SnehaKapoo1** [comment]
Ohh sorry, I didn't use WhatsApp, I can join on telegram?

**Comment by HeySitam** [comment]
Ok no issues. You can discuss here in the issue section also. Soon I will
create discord server. 😁

**Comment by SnehaKapoo1** [comment]
Yeah sure and could you please explain to me that In **About Section means up to? **

**Comment by HeySitam** [comment]
@SnehaKapoo1 open the nav drawer and here you will get the `-: About App :-` section.

**Comment by HeySitam** [comment]
@SnehaKapoo1 any update?
